app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,5 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredient.order(:name)
```

Comment:
We would love to see this `.order` encapsulated into a model method. Even though it's not a complex query, it's beyond basic CRUD methods that are appropriate for a controller (which are .create, .new, .save, .update, .find or .find_by)